package.json

```diff
@@ -10,33 +10,58 @@
   },
   "lint-staged": {
     "*.{js,mjs,jsx}": "eslint --cache --fix",
-    "*.{json,yml,css}": "prettier --write"
+    "*.{json,yml,css,scss}": "prettier --write"
   },
   "devDependencies": {
     "@babel/core": "^7.12.3",
     "@babel/eslint-parser": "^7.12.1",
+    "@babel/plugin-proposal-class-properties": "~7.12.1",
+    "@babel/plugin-transform-runtime": "~7.12.1",
+    "@babel/preset-env": "^7.12.1",
     "@babel/preset-react": "^7.12.1",
-    "@flybywiresim/rnp": "^1.1.1",
```

Comment:
This line should probably not be dropped.

Reply:
It's just moved below